tox.ini

```diff
@@ -31,6 +32,13 @@ changedir = tests
 commands =
     {envpython} runtests.py {posargs}
 
+[testenv:black]
+basepython = python3
+usedevelop = false
+deps = black
+changedir = {toxinidir}
+commands = black --check django tests
```

Comment:
We need to exclude `tests/test_runner_apps/tagged/tests_syntax_error.py`.

Reply:
and add `setup.py` and `scripts`

Reply:
Using `extend-exclude` (to respect `.gitignore`) and the using `.` rather than `django tests` gets all this: 

```diff
diff --git a/pyproject.toml b/pyproject.toml
index 9508ccc37f..f97c819c16 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -4,3 +4,5 @@ build-backend = 'setuptools.build_meta:__legacy__'
 
 [tool.black]
 target-version = ['py38']
+extend-exclude = 'tests/test_runner_apps/tagged/tests_syntax_error.py'
+
diff --git a/tox.ini b/tox.ini
index 9d4b6cab1c..49b22665de 100644
--- a/tox.ini
+++ b/tox.ini
@@ -37,7 +37,7 @@ basepython = python3
 usedevelop = false
 deps = black
 changedir = {toxinidir}
-commands = black --check django tests
+commands = black --check .
 
 [testenv:flake8]
 basepython = python3
```